### ISSUE TITLE
New problem in local deployment

### ISSUE THREAD

**Issue by ShengjieLuo** [pachyderm/pachyderm#778]
Hi! For some reasons, I deploy the pachyderm cluster in local server with current version. Some problems I meet in the deployment are list as following.

Kubernetes version:
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.2", GitCommit:"528f879e7d3790ea4287687ef0ab3f2a01cc2718", GitTreeState:"clean"}
Server Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.2", GitCommit:"528f879e7d3790ea4287687ef0ab3f2a01cc2718", GitTreeState:"clean"}
```

Pachctl version:
```
#pachctl version
COMPONENT           VERSION
pachctl             1.1.0-ba43f31be2be7e013f1373eac6581f9093f37f48
```

## Problem of make launch
Use `make launch` to launch the pachyderm, then we meet problems,
```
kubectl get all
NAME                   READY        STATUS             RESTARTS                       AGE
etcd-ukarn             1/1          Running            0                              4m
k8s-etcd-127.0.0.1     1/1          Running            0                              50m
k8s-master-127.0.0.1   4/4          Running            3                              51m
k8s-proxy-127.0.0.1    1/1          Running            0                              50m
pachd-40dqd            0/1          ImagePullBackOff   0                              4m
pachd-ic0zi            0/1          ImagePullBackOff   0                              4m
pachd-init-r4gb8       0/1          ImagePullBackOff   0                              4m
rethink-1ezrq          1/1          Running            0                              4m
```
`ImagePullBackOff ` lasts for a long time and no results.

## Use make launch-dev instead
Use `make launch-dev` to launch the pachyderm, we can deploy Pachyderm. But the version of `Pachd` and `Pachctl` is not the same.
```
pachctl version
COMPONENT           VERSION
pachctl             1.1.0-ba43f31be2be7e013f1373eac6581f9093f37f48
pachd               1.1.0
```
However, the pipeline does not work because of the version conflict.
```
./examples/fruit_stand/run.sh
+ pachctl create-repo data
+ pachctl start-commit data master
c4976921bab1419299fb37c76f063f70
+ cat examples/fruit_stand/set1.txt
+ pachctl put-file data master sales
+ pachctl finish-commit data master
+ pachctl create-pipeline -f examples/fruit_stand/pipeline.json
error from CreatePipeline: proto: bad wiretype for field pps.Transform.Debug: got wiretype 2, want 0
```
Unfortunately, the local Pachyderm deployment fails.

Turn the Pachctl version back to 1.1.0. The reason why I use `pachctl 1.1.0-ba43f31be2be7e013f1373eac6581f9093f37f48 ` before would be explained in the following comment.
```
COMPONENT           VERSION
pachctl             1.1.0
pachd               1.1.0
```
## Problems of example fruit-stand
RUN ` ./examples/fruit_stand/run.sh`. The example has a problem in JSON file.
```
+ pachctl create-repo data
+ pachctl start-commit data master
9c1cf5de91d0441682d32b7560beea0a
+ cat examples/fruit_stand/set1.txt+
pachctl put-file data master sales
+ pachctl finish-commit data master
+ pachctl create-pipeline -f examples/fruit_stand/pipeline.json
error marshalling JSON into protobuf: json: cannot unmarshal string into Go value of type bool
```
RUN `word-count` example. No problem.

## Suggestion for the problem
In conclusion, please recheck the following steps in deployment.

1. Fruit_stand example has a problem in JSON file.
2. Using `make launch-dev`  instead of `make launch` does not obey the mannual
3. The version problem of pachctl would be discussed in issue #779

**Comment by JoeyZwicker** [comment]
Sorry you ran into this. Yeah there are some version mismatch errors with
our examples right now, but our new docs site has all of these fixed and
will be released in the next 2 weeks.

**Comment by ShengjieLuo** [comment]
RE-test the Pachyderm with new master-version today. The version conflict and json problem has been solved. Thanks ! @jdoliner You can close the issue for it has been solved now.

**Comment by jdoliner** [comment]
Glad that worked!
